travelwise/frontend/templates/travel/create.html

```diff
@@ -42,13 +35,52 @@ <h2>Activities</h2>
     </div>
   </div>
 </main>
-
+{{activities|json_script:"activitiesJson"}}
 <script>
```

Comment:
Can you add the local script to its own file and add it to the bottom of this template?

Reply:
Do you mean the embedded json? Or are you talking about the inline script?